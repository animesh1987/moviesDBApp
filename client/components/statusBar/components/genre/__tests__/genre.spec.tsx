import * as React from 'react';
import { create } from 'react-test-renderer';

import { ByGenre } from '../index';

describe('Genre Test', () => {
  const genre = 'Test';

  it('renders correctly', () => {
    const component = create(<ByGenre genre={genre}/>)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});