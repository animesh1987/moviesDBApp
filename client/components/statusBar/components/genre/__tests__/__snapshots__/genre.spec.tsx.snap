// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Genre Test renders correctly 1`] = `
<span>
  Films by 
  Test
   genre
</span>
`;
